libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -188,8 +188,8 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @DisplayName: GPS update rate in milliseconds
     // @Description: Controls how often the GPS should provide a position update. Lowering below 5Hz(default) is not allowed. Raising the rate above 5Hz usually provides little benefit and for some GPS (eg Ublox M9N) can severely impact performance.
     // @Units: ms
-    // @Values: 100:10Hz,125:8Hz,200:5Hz
-    // @Range: 50 200
+    // @Values: 200:5Hz
+    // @Range: 200 200
```

Comment:
A range of 200 to 200 makes the param pointless for those who are less comfortable going outside the recommended ranges.

Reply:
@IamPete1 san. 
The explanation of this parameter states "Raising the rate above 5Hz usually provides little benefit".
I understand that ArduPilot does not recommend anything other than 5Hz.
It also states "some GPS (eg Ublox M9N) can severely impact performance".
I would like to see a warning message if I try to set a period that is not recommended.